docker-compose.dev.yml

```diff
@@ -30,6 +30,8 @@ services:
       WEB_URL: ${WEB_URL}
       REFRESH_TOKEN: ${REFRESH_TOKEN}
       EMAIL: ${EMAIL}
+      GITHUB_CLIENT_ID: ${GITHUB_CLIENT_ID}
+      GITHUB_CLIENT_SECRET: ${GITHUB_CLIENT_SECRET}
```

Comment:
Didi you forget `BASE_URL: ${BASE_URL}`  here?

Reply:
fixed